src/Components/RegistrationView/RegistrationView.js

```diff
@@ -87,11 +88,14 @@ function RegistrationView() {
               formData={registrationData}
             />
           )}
+          {currentStep === 4 && <FifthStep currentStep={currentStep} formData={registrationData} onSubmit={() => goToLogin()}/>}
         </div>
-        <div className='registration__sign-in'>
+        {currentStep !== 4 && 
+<div className='registration__sign-in'>
```

Comment:
Hmmm looks like wrong indent. Does your eslint work?

Reply:
It is weird as I have done nothing but this place is fixed and looks correctly. I guess, there was temporary bug in eslint or something around it.